x/slashing/genesis.go

```diff
@@ -5,10 +5,32 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/stake/types"
 )
 
-// InitGenesis initializes the keeper's address to pubkey map.
-func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) {
-	for _, validator := range data.Validators {
+// GenesisState - all slashing state that must be provided at genesis
+type GenesisState struct {
+	Params Params
+}
+
+// HubDefaultGenesisState - default GenesisState used by Cosmos Hub
+func HubDefaultGenesisState() GenesisState {
```

Comment:
I think this should just be renamed to DefaultGenesisState, we expect other people to be able to use the same api and etc.